Dockerfile

```diff
@@ -1,15 +1,15 @@
 # Build container
 FROM node:6-alpine as build_container
 
-WORKDIR /home/node
+WORKDIR /root
```

Comment:
Keep all the build in `/home/node`: building and running is a different matter, it should not affect the way you start the application in the end and it avoid most of the fuzz of your PR.